Source/MQTTnet/Client/Options/MqttClientOptionsBuilder.cs

```diff
@@ -456,6 +456,12 @@ public MqttClientOptionsBuilder WithWillRetain(bool willRetain = true)
             return this;
         }
 
+        public MqttClientOptionsBuilder WithWillMessageExpiryInterval(uint willMessageExpiryInterval)
```

Comment:
I would prefer the XML doc way because we do not have any prefixes etc. in the project at all. 
We may also add an overload which accepts a TimeSpan so that the user can safely define the value and we make use of the total seconds (rounded).

Reply:
But for this PR it is fine in my opinion.